plugins/sinks/kafka/sink.go

```diff
@@ -2,79 +2,68 @@ package kafka
 
 import (
 	"context"
-	"encoding/json"
-	"errors"
-	"fmt"
-
 	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/golang/protobuf/proto"
 	"github.com/odpf/meteor/core"
 	"github.com/odpf/meteor/core/sink"
+	"github.com/odpf/meteor/utils"
+	"github.com/pkg/errors"
 )
 
 type Sink struct{}
 
-func init() {
-	if err := sink.Catalog.Register("kafka", New()); err != nil {
-		panic(err)
-	}
-}
-
 func New() core.Syncer {
 	return new(Sink)
 }
 
-// Kafka sink
-func (s *Sink) Sink(ctx context.Context, config map[string]interface{}, out <-chan interface{}) (err error) {
+type Config struct {
+	Brokers string `mapstructure:"brokers"`
```

Comment:
super sorry for not being clear, you need to define if the field if required using tag like
```
Brokers string `mapstructure:"brokers" validate: "required"`
```

Sample:
https://github.com/odpf/meteor/blob/main/plugins/extractors/mysql/extractor.go#L23